docs/detections/value-list-exceptions.asciidoc

```diff
@@ -37,27 +37,54 @@ act as delimiters.
 =========================
 
 . Go to *Rules* -> *Detection rules (SIEM)*.
-. Click *Import value lists*. The *Import value lists* window opens.
+. Click *Manage value lists*. The *Manage value lists* window opens.
 +
 [role="screenshot"]
-image::images/upload-lists-ui.png[Import value lists flyout,75%]
+image::images/upload-lists-ui.png[Manage value lists flyout,75%]
 
 . Select the list type (*Keywords*, *IP addresses*, *IP ranges*, or *Text*) from the *Type of value list* drop-down.
 . Drag or select the `csv` or `txt` file that contains the values.
 . Click *Import value list*.
 
-NOTE: When the name of the file you are importing already exists, the values in
-the new file are appended to the previously imported values.
+NOTE: If you import a file with a name that already exists, a new list is not created. The imported values are added to the existing list instead.
 
-[[edit-value-lists]]
+[[manage-value-lists]]
 [discrete]
 === Manage value lists
 
-To view, delete, or export existing value lists:
+You can edit, remove, or export existing value lists. 
+
+[[edit-value-lists]]
+[discrete]
+==== Edit value lists
+
+. Go to **Rules** → **Detection rules (SIEM)**.
+. Click **Manage value lists**. The **Manage value lists** window opens. 
+. In the **Value lists** table, click the value list you want to edit.
+. Do any of the following:
+
+** **Filter items in the list**: Use the KQL search bar to find values in the list. Depending on your list's type, you can filter by the `keyword`, `ip_range`, `ip`, or `text` fields. For example, to filter by Gmail addresses in a value list of the `keyword` type, enter `keyword:*gmail.com` into the search bar. 
++
+You can also filter by the `updated_by` (for example, `updated_by:testuser`), or `updated at` (for example, `updated_at < now`) fields.  
+** **Add individual items to the list**: Click **Create list item**, enter a value, then click **Add list item**.
+** **Bulk upload list items**: Drag or select the `csv` or `txt` file that contains the values that you want to add, then click **Upload**.
+** **Edit a value**: In the Value column, go to the value you want to edit and click the **Edit** button (image:images/edit-value-list-item.png[Edit button from Manage value lists window,15,15]). When you're done editing, click the **Save** button (image:images/save-value-list-item-changes.png[Save button from Manage value lists window,18,18]) to save your changes. Click the **Cancel** button (image:images/cancel-value-list-item-changes.png[Cancel button from Manage value lists window,18,18]) to revert your changes. 
+** **Remove a value**: Click the **Remove value list** button (image:images/remove-value-list-item.png[Remove value list button from Manage value lists window,15,15]) to delete a value from the list.
```

Comment:
```suggestion
** **Remove a value**: Click the **Remove value** button (image:images/remove-value-list-item.png[Remove value list button from Manage value lists window,15,15]) to delete a value from the list.
```
Not familiar with the 8.14 UI but I thought this is just for removing a single value, not a list.